### PR TITLE
Fix issues with  options

### PR DESCRIPTION
**Pull request by petergebala**
## What
In some cases we get exception `no block given`. It should fix it.

### CODE REVIEW

File: spec/opera/operation/base_spec.rb [hunk 1]
```diff
@@ -559,6 +559,31 @@ def step_2
         end
       end
 
+      context 'for accessors/readers' do
+        subject { operation_class.call(params: {}, dependencies: {}) }
+
+        let(:operation_class) do
+          Class.new(Operation::Base) do
+            params_reader :bar, default: -> { :bar }
```
Comment (BravoSimone) [inline spec/opera/operation/base_spec.rb:567]: @petergebala I think this spec isn't covering this fix properly. I was trying to check it locally and if the param has a default the issue is not happening. Do you mind adding another param reader or another context to cover this?

**Review by BravoSimone — CHANGES_REQUESTED** [review]

File: spec/opera/operation/base_spec.rb (hunk 1 above)
Comment (petergebala) [reply spec/opera/operation/base_spec.rb:567]: @BravoSimone was right. I missed one change in specs. Fixed and pushed with force

**Review by BravoSimone — APPROVED** [review]
🚀

**Review by no17436 — APPROVED** [review]

**Comment by kikorb** [comment]
Too fast I was going to suggest that we drop default -> in favour of default: & eval: -> and maintain default: -> and deprecate it soon

**Pull request merged**